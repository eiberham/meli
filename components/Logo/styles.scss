.logo {
    padding: 0 1rem 0 0;
    &:hover {
        cursor: pointer;
    }
}