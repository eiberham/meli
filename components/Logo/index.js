import React from 'react';
import './styles.scss';

export default function (){
    return (
        <div className="logo">
            <img src="/static/Logo_ML.png" alt="logo" />
        </div>
    )
}