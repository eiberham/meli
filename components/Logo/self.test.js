import React from 'react';
import renderer from 'react-test-renderer';
import Logo from './index';

describe('Logo', () => {
    it('matches snapshot', () => {
        const component = renderer.create(<Logo />).toJSON();
        expect(component).toMatchSnapshot();
    })
})