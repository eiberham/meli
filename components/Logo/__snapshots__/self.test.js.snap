// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo matches snapshot 1`] = `
<div
  className="logo"
>
  <img
    alt="logo"
    onClick={[Function]}
    onMouseEnter={[Function]}
    src="/static/Logo_ML.png"
  />
</div>
`;
